### DISCUSSION TITLE
How to change the tab icon?

### DISCUSSION THREAD

**Discussion by MikiVanousek** [jpanther/congo#119] [Q&A] [answered]
Greetings!
Foremost, what an amazing theme! Thank you for making it. :heart: 

I switched to the `slate` color scheme, but the icon in the tab is still purple. Is there a way to change it?
![Image](https://user-images.githubusercontent.com/33402194/153821055-42a6f4f7-8675-4463-bee5-8c601a623150.png)

**Comment by cbelena** [comment, marked as answer]
Hi! Yes, that's the favicon. You can change it easily. Check out [this section in the docs](https://jpanther.github.io/congo/docs/partials/#favicons).

**Reply by MikiVanousek** [reply]
Thanks. I did not know what a favicon is, so I did not realize this when reading the section.
